Cargo.toml

```
@@ -29,6 +29,7 @@ cid = { default-features = false, version = "0.5" }
 trust-dns-resolver = "0.20"
 either = { default-features = false, version = "1.5" }
 futures = { default-features = false, version = "0.3.9", features = ["alloc", "std"] }
+hash_hasher = { default-features = false, version = "2.0.3" } 
```

```suggestion
hash_hasher = "2.0.3"
```

Since I happened to look, the crate defines no features. I wish there was a lint for these.

---

Many thanks for spotting this, I didn't check if the crate had any features in the first place.